libs/coin-polkadot/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./test";
```

Comment:
I think in future, we want to stop bundling testing data in our production builds, it could be good to start making this separation, meaning, maybe the test is one of these "exception" that is not part of the "bundle". becuase it make the end bundle heavier and it's only for tests

Reply:
I fully agree. But I lack of knowledge on npm to have a clean idea on how it could work.